pkg/healthcheck/healthcheck.go

```diff
@@ -450,6 +451,11 @@ func NewHealthChecker(categoryIDs []CategoryID, options *Options) *HealthChecker
 	return hc
 }
 
+func NewWithBasicCategories(options *Options) *HealthChecker {
```

Comment:
nit: how do you feel about `NewWithCoreChecks`, bit shorter, arguably a little more descriptive. It's okay not to make the change :D